editor/animation_track_editor.cpp

```diff
@@ -2695,18 +2695,25 @@ void AnimationTrackEdit::gui_input(const Ref<InputEvent> &p_event) {
 					AnimationPlayer *ap = ape->get_player();
 					if (ap) {
 						NodePath npath = animation->track_get_path(track);
-						Node *nd = ap->get_node(ap->get_root_node())->get_node(NodePath(npath.get_concatenated_names()));
-						StringName prop = npath.get_concatenated_subnames();
-						PropertyInfo prop_info;
-						ClassDB::get_property_info(nd->get_class(), prop, &prop_info);
+						Node *aApRootNode = ap->get_node(ap->get_root_node());
```

Comment:
```suggestion
						Node *a_ap_root_node = ap->get_node(ap->get_root_node());
```
Use `snake_case`

Reply:
my bad I'm too used to camel case from my workplace, I'm gonna fix this